src/MonologCreator/Factory.php

```diff
@@ -130,6 +130,13 @@ public function createProcessors(array $loggerConfig)
                 $processors[] = $webProcessor;
             } elseif ('requestId' === $processor) {
                 $processors[] = new Processor\RequestId();
+            } elseif ('extraField') {
```

Comment:
I think this was suppose to be a comparison.

Reply:
Exactly. Will fix!